src/View/XmlView.php

```diff
@@ -95,9 +95,9 @@ class XmlView extends SerializedView
      *   'format' as 'attributes' instead of 'tags'.
      *
      * @param array|string|true $serialize The name(s) of the view variable(s) that need(s) to be serialized
-     * @return string The serialized data
+     * @return string|false The serialized data
      */
-    protected function _serialize($serialize): string
+    protected function _serialize($serialize)
```

Comment:
This will compromise the render() part (typehinted return string there) and other higher levels. Why not catching the false and returning empty string insdead?

Reply:
`Serialized::render()` throws exception if this method returns false, so there's no comprise at the level of public API. Allowing this method to return false was just convenient as that's the return value of `json_encode()` in case of failure.

Reply:
Wouldnt it be cleaner/clearer to let this method blow up?
If somewhere later the false leads to an issue it is harder to track down, the cause is cloaked then.

I am for more defensive coding in the right places, and meaningful exceptions including the correct stack trace then. This also allows us to keep the typehints that are supposed to document the code :)